src/context/Global/index.tsx

```diff
@@ -233,6 +233,10 @@ const GlobalProvider = ({ children }: { children: ReactNode }) => {
     return undefined;
   };
 
+  const getFirstSolutionVariant = (): Table => {
+    return remainingSolutionVariants[0];
```

Comment:
Why use remainingSolutionVariants instead of allSolutionVariants

Reply:
To be honest, I was not able to fully grasp the difference between the two. `remainingSolutionVariants` seems to be filtered in `trimSolutions()` but I haven't further looked into by what exactly. I just thought it'd be more accurate to use here.

Reply:
For any given guess the 3x3 grid is first checked if they got the full correct recipe, so the guess is checked against all the possible solution variants (`allSolutionVariants`).

If they didn't get the full correct recipe, then we start checking if they got any of the items in the right slots so we can show the yellow and green cells as hints. Since some recipes have many different possible positions, we need to avoid showing inconsistent hints

e.g. imagine the solution is a stick.  In the their first guess they guess (W is wood):

```
W X X
X X X
X X X
```

So the hint shows green in the top left corner.  Then on their next guess 

```
X X X
X X X
X X W
```

We don't want to show green in the bottom left because that would be very confusing since we've already said that there is wood in the top left corner, even though bottom left is also technically a possible spot.

remainingSolutionVariants is the list of solutions that are still possible given the green correct cells the user has been shown.

Reply:
This code is fine, I was just wondering why you chose this.

Reply:
Oh, I understand now. I haven't noticed this behaviour when playing before. Thanks!